CMSgov/mint-app
# [EASI-4280] Addressing text overrunning

# EASI-4280


## Changes and Description

- Add new custom style to use `overflow-wrap` property
- Add classname to where it is necessary



## How to test this change

1. Add the following text to Model Basic's Problem statement, Test Interventions, and Notes
```
This model will pair existing activities (e.g. birthing-friendly hospital designations and 12-month postpartum coverage) with new activities that are supported through both the technical assistance and CMMI model (e.g. updated reimbursement rates and robust screening and referral networks). Hallmarks of successful care transformation -- which will ultimately inform the model's metrics of success -- will be provider continuity, relationship-based care, and resolution of comprehensive needs (clinical, behavioral, oral, and social) for pregnant and postpartum Medicaid beneficiaries. We anticipate that this transformation will lead to reductions in medical conditions (e.g. pregnancy hypertension, gestational diabetes)[i] that drive morbidity and mortality, appropriate management of chronic / emergent conditions, reduction in unnecessary cesarean section, and increased identification and treatment of maternal depression and anxiety.[ii][i]https://www.liebertpub.com/doi/abs/10.1089/jwh.2020.8464[ii]https://womensmentalhealth.org/posts/maternal-anxiety-and-adverse-perinatal-outcomes-why-we-need-to-screen-women-for-anxiety-during-pregnancy/
```
2. Save and go to the read only view
3. Verify that the text is not overrunning 

### Before 

![image](https://github.com/CMSgov/mint-app/assets/2349702/3a5422a9-c752-48ed-bc3b-f87dee99bcac)

### After

![image](https://github.com/CMSgov/mint-app/assets/2349702/f9eb1153-3cdf-46ae-8b0c-312f1d31bb62)








## PR Author Review Checklist



## PR Reviewer Guidelines
- It's best to pull the branch locally and test it, rather than just looking at the code online!
- Check that all code is adequately covered by tests - if it isn't feel free to suggest the addition of tests.
- Always make comments, even if it's minor, or if you don't understand why something was done.
